tools/querytee/proxy_endpoint_test.go

```diff
@@ -353,67 +354,81 @@ func Test_ProxyEndpoint_Comparison(t *testing.T) {
 func Test_ProxyEndpoint_LogSlowQueries(t *testing.T) {
 	scenarios := map[string]struct {
 		slowResponseThreshold         time.Duration
-		preferredResponseLatency      time.Duration
-		secondaryResponseLatency      time.Duration
+		latencyPairs                  []latencyPair
```

Comment:
Is this change necessary? The previous setup seems clearer to me, and we can construct the `[]time.Duration` slices for `newMockProxyBackend` directly (rather than using `splitLatencyPairs`)

Reply:
This lets us create multiple data points for the histogram. Possibly overkill, but it's here now, so unless you think it's overly convoluted I'd say we leave it in case we need it for future tests.

Reply:
For `Test_ProxyEndpoint_RelativeDurationMetric`, I think using `latencyPairs` like this makes sense, but for this test (`Test_ProxyEndpoint_LogSlowQueries`), it obscures the intent - we never intend to test multiple requests in this test.

Reply:
There is no reason why `Test_ProxyEndpoint_LogSlowQueries` couldn't test multiple points though. It also verifies the number of points with `waitForResponseComparisonMetric` which takes in the expected count.

I can change it back if you think it's a big issue, but in my opinion this makes the mock asset more flexible for future tests etc. and would rather not waste too many cycles here.